README.md

```diff
@@ -24,6 +25,22 @@ Naming things is hard. This sheet attempts to make it easier.
 
 Although these suggestions can be applied to any programming language, I will use JavaScript to illustrate them in practice.
 
+## English language
+
+Use English language when naming your variables and functions.
+
+```js
+/* Bad */
+const primerNombre = 'Gustavo'
+const amigos = ['Kate', 'John']
+
+/* Good */
+const firstName = 'Gustavo'
+const friends = ['Kate', 'John']
+```
+
+> Like it or not, English is the dominant language in programming: the syntax of all programming languages is written in English, as well as countless documentations and educational materials. By writing your code in English you dramatically increase its cohesiveness.
```

Comment:
🚀

Reply:
@Aprillion, thank you for mentioning this. This has been a huge pain point in my experience and it's definitely worth mentioning as a guideline to follow.